backend/routes/db.py

```diff
@@ -622,6 +622,11 @@ def n3c_recommended_report(as_json=False) -> Union[List[str], Dict]:
 
 
 @router.get("/n3c-comparison-rpt")
+def _n3c_comparison_rpt():
+    return n3c_comparison_rpt()
+
+
+# @cache
```

Comment:
I wrote that I added caching but it looks like I commented it out because it wasn't the cause of the problem we were experiencing.